package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "manh-react-survey",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "axios": "^0.21.1",
+    "i18next": "^21.6.14",
+    "i18next-browser-languagedetector": "^6.1.3",
+    "i18next-http-backend": "^1.4.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-i18next": "^11.16.1",
+    "react-router-dom": "^6.3.0",
+    "react-scripts": "5.0.1",
+    "sass": "^1.49.11",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts -r @cypress/instrument-cra start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "test:coverage": "rimraf coverage && react-scripts test --coverage --watchAll=false && npm run cypress:run && node ./scripts/coverage-merge.js && nyc report",
+    "test:merge-coverage": "node ./scripts/coverage-merge.js",
+    "lint": "eslint ./ --ext .js,.ts,.tsx",
+    "lint:fix": "eslint ./ --ext .js,.ts,.tsx --fix",
+    "stylelint": "stylelint '**/*.scss'",
+    "stylelint:fix": "stylelint '**/*.scss' --fix",
+    "codebase:lint": "npm run lint && npm run stylelint",
+    "codebase:fix": "npm run lint:fix && npm run stylelint:fix",
+    "cypress": "start-server-and-test start 3000 cypress:run",
+    "cypress:run": "cypress run",
+    "cypress:open": "cypress open"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "devDependencies": {
+    "@cypress/code-coverage": "^3.9.12",
+    "@cypress/instrument-cra": "^1.4.0",
+    "@nimblehq/eslint-config-nimble-react": "^1.2.0",
```

Comment:
let's add `@nimblehq/eslint-config-nimble-testing` for testing purpose linter as well
![image](https://github.com/manh-t/react-survey/assets/42540264/401446f4-d277-4061-b57a-2eadc053042a)

FYI: https://github.com/nimblehq/eslint-config-nimble/tree/develop/packages/eslint-config-nimble-react

Reply:
@nvminhtue added in [b68ff78](https://github.com/manh-t/react-survey/pull/39/commits/b68ff7877896343f127df50578d91e3d4360f9f1)